src/server/api.php

```diff
@@ -34,14 +34,29 @@
 
 // passwords
 $AUTH = array(
-  'twitter'  => Secrets::$api['twitter'],
-  'ght'      => Secrets::$api['ght'],
-  'fluview'  => Secrets::$api['fluview'],
-  'cdc'      => Secrets::$api['cdc'],
-  'sensors'  => Secrets::$api['sensors'],
-  'quidel'   => Secrets::$api['quidel'],
-  'norostat' => Secrets::$api['norostat']
+  'twitter'        => Secrets::$api['twitter'],
+  'ght'            => Secrets::$api['ght'],
+  'fluview'        => Secrets::$api['fluview'],
+  'cdc'            => Secrets::$api['cdc'],
+  'sensors'        => Secrets::$api['sensors'],
+  'sensor_subsets' => Secrets::$api['sensor_subsets'],
+  'quidel'         => Secrets::$api['quidel'],
+  'norostat'       => Secrets::$api['norostat'],
 );
+$GRANULAR_SENSOR_AUTH_TOKENS = array(
+  'twtr' => array($AUTH['sensor_subsets']['twtr_sensor']),
+  'gft' => array($AUTH['sensor_subsets']['gft_sensor']),
+  'ght' => array($AUTH['sensor_subsets']['ght_sensors']),
+  'ghtj' => array($AUTH['sensor_subsets']['ght_sensors']),
```

Comment:
I don't see a corresponding entry in the operations repo for ghtj. Is that one used anywhere? Justin made it a long time ago, but I'm not sure if it has been used since.

Reply:
The 'ght_sensors' token gives access to both GHT-derived sensor sets: ght and ghtj.  I'm not clear if ghtj is being used in the nowcasts yet.

Reply:
Ah ok, that makes sense.